README.md

```diff
@@ -23,5 +23,6 @@ Roadmap
 
 Known Bug Workarounds
 =====================
-* Burrito launches off-center or on the wrong monitor
-  * This seems to be WM dependant. on Gnome it can be solved by holding down the super key and dragging burrito to the right position.
+* Burrito launches off-center or on the wrong monitor (This seems to be WM dependant.)
+  * On Gnome and KDE it can be solved by holding down the super key and dragging burrito to the right position.
+  * On KDE it is also possible to use window-rules to set the position or a target monitor automatically.
```

Comment:
Can we insert a link to documentation or include some instructions as to how window-rules works? I want to make sure folks reading the docs either know what they need to know, or know where to look next.

Otherwise this looks good.

Reply:
Understood.
I didn't want to overdo it, because Gnome is the official target DE.
I've added some useful information about the KDE window management tools.